output/grainIntegration/2022-04-24T00:00:00.000Z_Payouts-1653034400307_parcel.csv

```diff
@@ -1,20 +1,15 @@
-Name(Optional),Address/ENS,Amount,Token Address/Token Symbol
+Name(Optional),receiver,amount,token_address
```

Comment:
Can we keep the original `Name(Optional),Address/ENS,Amount,Token Address/Token Symbol` column names instead of renaming them?

Here is the sample CSV that Parcel shared in their website: https://docs.google.com/spreadsheets/d/1mHPayaGG10gwmtbNVeDig_oNBfOwLQ6S_JaiDZo5io4/edit?usp=sharing

![Screen Shot 2022-07-07 at 3 28 48 PM](https://user-images.githubusercontent.com/95955389/177716886-f5b4a726-3ac4-4007-a545-96a524eff433.png)